runtime-management/runtime-management-impl/src/main/resources/main.py

```diff
@@ -66,13 +67,17 @@ def main(self):
                 final_result = self.__process_result(result)
             finally:
                 self.__enable_standard_io(old_io)
-        except Exception as e:
-            exc_tb = sys.exc_info()[2]
-            final_result = {
-                "exception": str(e),
-                "traceback": traceback.format_list(traceback.extract_tb(exc_tb))
-            }
-
+        except ExecutionException as e:
```

Comment:
This doesn't seem right. You should have two different except blocks:
`except ExecutionException as e:` with it's code and `except Exception as e:` with the default code. All this instead of the if else.
See here https://stackoverflow.com/questions/6095717/python-one-try-multiple-except/6095782